bench/hashing_running_time.rs

```diff
@@ -0,0 +1,68 @@
+use cairo_vm::hint_processor::hint_processor_definition::HintProcessorLogic;
+use cairo_vm::{
+    hint_processor::builtin_hint_processor::builtin_hint_processor_definition::HintProcessorData,
+    types::exec_scope::ExecutionScopes,
+    vm::{runners::cairo_runner::RunResources, vm_core::VirtualMachine},
+};
+use criterion::{criterion_group, criterion_main, Criterion};
+use starknet_in_rust::syscalls::deprecated_syscall_handler::DeprecatedSyscallHintProcessor;
+use starknet_in_rust::{
+    state::{cached_state::CachedState, in_memory_state_reader::InMemoryStateReader},
+    syscalls::deprecated_business_logic_syscall_handler::DeprecatedBLSyscallHandler,
+};
+use std::collections::HashMap;
+use std::{any::Any, hint::black_box};
+
+#[allow(dead_code)]
+fn criterion_benchmark(c: &mut Criterion) {
+    let exec_scopes = &mut ExecutionScopes::new();
+    let ids_names = vec!["syscall_ptr"];
+    let references = {
+        let mut references = HashMap::<
+            usize,
+            cairo_vm::hint_processor::hint_processor_definition::HintReference,
+        >::new();
+        for i in 0..ids_names.len() as i32 {
+            references.insert(
+                i as usize,
+                cairo_vm::hint_processor::hint_processor_definition::HintReference::new_simple(i),
+            );
+        }
+        references
+    };
+
+    let mut ids_data =
+        HashMap::<String, cairo_vm::hint_processor::hint_processor_definition::HintReference>::new(
+        );
+    for (i, name) in ids_names.iter().enumerate() {
+        ids_data.insert(name.to_string(), references.get(&i).unwrap().clone());
+    }
+
+    let hint_data: Box<dyn Any> = Box::new(HintProcessorData::new_default(
+        "syscall_handler.get_block_timestamp(segments=segments, syscall_ptr=ids.syscall_ptr)"
+            .to_string(),
+        ids_data,
+    ));
```

Comment:
We can probably iterate through the keys and values of the hint code map and use the key as code (as that's what it is) and the value formatted as string as the name (it will print as `HintCode::HintName` or something like that). I think it will lead to simpler code and to testing all the different hints.